ext/http/lib.rs

```diff
@@ -164,7 +164,7 @@ impl HttpConnResource {
   // Accepts a new incoming HTTP request.
   async fn accept(
     self: &Rc<Self>,
-  ) -> Result<Option<HttpStreamResource>, AnyError> {
+  ) -> Result<Option<(HttpStreamResource, String, String)>, AnyError> {
```

Comment:
This would have been nicer as a struct instead of a tuple, but not the end of the world :)